utxo/Cargo.toml

```diff
@@ -13,5 +13,5 @@ logging = { path = "../logging/" }
 serialization = { path = "../serialization" }
 
 parity-scale-codec = { version = "3.1.2", features = ["chain-error"] }
-iter_tools = "0.1.0"
+itertools = "0.10"
```

Comment:
why `0.10` over the newest version?

Reply:
The `0.10` version is anything `0.10.0 <= x < 0.11`, so this will match the newest version `0.10.3`, which is nailed down in `Cargo.lock`. Some [docs here](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio).

Reply:
You know what I completely misread that. Ignore me entirely

Reply:
> Without a patch version specified the dependency resolver will take the latest one (or a version compatible to the specified for other crates). We already have the Cargo.lock file committed in the repository, so one less reason to stick to full versions.

Yeah I agree. Just ignore me. I completely misread it in the first place